.github/workflows/ci.yml

```diff
@@ -1,76 +1,79 @@
 name: CI
 
-# These trigger patterns courtesy of https://github.com/broccolijs/broccoli/pull/436
 on:
-  pull_request:
   push:
-    # filtering branches here prevents duplicate builds from pull_request and push
     branches:
       - main
-      - "v*"
-    # always run CI for tags
-    tags:
-      - "*"
+      - master
+  pull_request: {}
 
-env:
-  CI: true
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
 
 jobs:
-  test-locked-deps:
-    name: Locked Deps
+  test:
+    name: "Tests"
     runs-on: ubuntu-latest
+    timeout-minutes: 10
+
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v3
       - name: Install Node
-        uses: actions/setup-node@v2.1.5
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 12.x
```

Comment:
I see that `v5` of the addon supports Node 10.

I wonder if we can skip to supporting Node 14 for the next major version release, as Node 12 is no longer LTS. What are your thoughts?

Reply:
👍 from my side. @tehhowch what do you think?

Reply:
whatever the minimum node supported by ember-source v3.28.0 is, should be the goal Reading [this](https://github.com/emberjs/ember.js/blob/release-3-28/package.json) and the [v4 release notes](https://github.com/emberjs/ember.js/releases/tag/v4.0.0) suggest node 10 is the minimum supported by 3.28

Reply:
Ember cli [needs](https://github.com/ember-cli/ember-cli/blob/v3.28.5/package.json#L167-L169) 12 so I'll leave as is